### PR TITLE
VCS Event Viewer

### PR DESCRIPTION
**Pull request by stasquatch**
## Labels

- [ ] inaccuracy
- [ ] clarification
- [x] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout

### CODE REVIEW

File: content/source/docs/cloud/api/vcs-events.html.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+layout: "cloud"
+page_title: "VCS Events - API Docs - Terraform Cloud and Terraform Enterprise"
+---
+
+# VCS Events API
+
+-> **Note**: The VCS Events API is still in beta as support while added for additional VCS providers. Currently only GitLab.com connections established after December 2020 are supported.
```
Comment (vaficionado) [inline content/source/docs/cloud/api/vcs-events.html.md:8]:
```suggestion
-> **Note**: The VCS Events API is still in beta as support is being added for additional VCS providers. Currently only GitLab.com connections established after December 2020 are supported.
```

Comment (stasquatch) [inline content/source/docs/cloud/api/vcs-events.html.md:8]:
```suggestion
-> **Note**: The VCS Events API is still in beta while support is added for additional VCS providers. Currently only GitLab.com connections established after December 2020 are supported.
```

File: content/source/docs/cloud/vcs/index.html.md [hunk 2]
```diff
@@ -98,3 +98,15 @@ For complete details, click the link for your VCS provider:
 - [Azure DevOps Services](./azure-devops-services.html)
 
 -> **Note:** Alternately, you can skip the OAuth configuration process and authenticate with a personal access token. This requires using Terraform Cloud's API. For details, see [the OAuth Clients API page](../api/oauth-clients.html).
+
+## Viewing events
+
+-> **Note**: The VCS Event Viewer is still in beta while support is added for additional VCS providers. Currently only GitLab.com connections established after December 2020 are supported.
```
Comment (stasquatch) [inline content/source/docs/cloud/vcs/index.html.md:104]:
```suggestion
-> **Note**: The VCS Events API is still in beta while support is added for additional VCS providers. Currently only GitLab.com connections established after December 2020 are supported.
```

File: content/source/docs/cloud/vcs/index.html.md [hunk 3]
```diff
@@ -98,3 +98,15 @@ For complete details, click the link for your VCS provider:
 - [Azure DevOps Services](./azure-devops-services.html)
 
 -> **Note:** Alternately, you can skip the OAuth configuration process and authenticate with a personal access token. This requires using Terraform Cloud's API. For details, see [the OAuth Clients API page](../api/oauth-clients.html).
```
Comment (vaficionado) [inline content/source/docs/cloud/vcs/index.html.md:100]:
```suggestion
-> **Note:** Alternatively, you can skip the OAuth configuration process and authenticate with a personal access token. This requires using Terraform Cloud's API. For details, see [the OAuth Clients API page](../api/oauth-clients.html).
```

Comment (vaficionado) [reply content/source/docs/cloud/vcs/index.html.md:100]: https://grammarist.com/usage/alternately-vs-alternatively/

File: content/source/docs/cloud/vcs/index.html.md [hunk 4]
```diff
@@ -98,3 +98,15 @@ For complete details, click the link for your VCS provider:
...
+
+## Viewing events
+
+-> **Note**: The VCS Event Viewer is still in beta while support is added for additional VCS providers. Currently only GitLab.com connections established after December 2020 are supported.
+
+VCS events describe changes within your organization for VCS-related actions, such as creating a new VCS provider (also called OAuth Client), loading repositories while creating a new workspace, or processing incoming webhooks. Events are limited to the past 10 days.
```
Comment (vaficionado) [inline content/source/docs/cloud/vcs/index.html.md:106]:
```suggestion
VCS events describe changes within your organization for VCS-related actions, such as creating a new VCS provider (also called an OAuth Client), loading repositories while creating a new workspace, or processing incoming webhooks. Displayed events are limited to the past 10 days.
```

**Review by vaficionado — COMMENTED** [review]
Couple small suggestions, otherwise LGTM

File: content/source/docs/cloud/vcs/index.html.md [hunk 5]
```diff
@@ -97,4 +97,16 @@ For complete details, click the link for your VCS provider:
 - [Azure DevOps Server](./azure-devops-server.html)
 - [Azure DevOps Services](./azure-devops-services.html)
 
--> **Note:** Alternately, you can skip the OAuth configuration process and authenticate with a personal access token. This requires using Terraform Cloud's API. For details, see [the OAuth Clients API page](../api/oauth-clients.html).
+-> **Note:** Alternatively, you can skip the OAuth configuration process and authenticate with a personal access token. This requires using Terraform Cloud's API. For details, see [the OAuth Clients API page](../api/oauth-clients.html).
+
+## Viewing events
+
+-> **Note**: The VCS Event Viewer is still in beta while support is added for additional VCS providers. Currently only GitLab.com connections established after December 2020 are supported.
```
Comment (stasquatch) [inline content/source/docs/cloud/vcs/index.html.md:104]:
```suggestion
-> **Note**: The VCS Event Viewer is still in beta as support is being added for additional VCS providers. Currently only GitLab.com connections established after December 2020 are supported.
```

**Pull request merged**